lib/volk_malloc.c

```diff
@@ -50,6 +50,10 @@
 
 void* volk_malloc(size_t size, size_t alignment)
 {
+    // Round size to a multiple of the alignment
+    if (size % alignment) {
+        size += alignment - (size % alignment);
+    }
```

Comment:
I'd argue the code is pretty clear. And does not require this comment.

However, I suggest to write a comment that explains the reason to do this change.
```
// Make ASAN happy...
```